sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -118,12 +120,27 @@ export function useFormatters() {
     };
   };
 
+  /**
+   * Convert a string or date (Z  OR 00:00 or 0 offset or date only)
+   * to a string format like "Aug 05 2021".
+   * @param date string or date to be converted.
+   * @returns string representation (e.g. Aug 05 2021).
+   */
+  const originalDateOnlyLongString = (date: string | Date): string => {
+    if (date) {
+      return dayjs(date)
+        .utc()
+        .format("MMM D, YYYY");

```

Comment:
This format **MMM D, YYYY** can be taken from a constant. Why I mentioned this is because if we have all the date formats in one place, then migration of date frameworks will be more adaptable.